joern-cli/frontends/pysrc2cpg/src/main/scala/io/joern/pysrc2cpg/NodeBuilder.scala

```diff
@@ -74,8 +74,21 @@ class NodeBuilder(diffGraph: DiffGraphBuilder) {
     addNodeToDiff(memberNode)
   }
 
+  def memberNode(name: String, lineAndColumn: LineAndColumn): nodes.NewMember = {
```

Comment:
This could be simplified to 
```scala
def memberNode(name: String, lineAndColumn: LineAndColumn): nodes.NewMember =
     memberNode(name)
       .lineNumber(lineAndColumn.line)
       .columnNumber(lineAndColumn.column)
 ```

Reply:
Udpated